src/shared/components/table/TableComponent.tsx

```diff
@@ -72,11 +72,11 @@ const TableComponent: React.FC<React.PropsWithChildren<TableProps>> = ({
 }) => {
   const filters = useDeepCompareMemoize(initFilters);
   const Header = useDeepCompareMemoize(initHeader);
-  //const [, setWindowWidth] = React.useState(window.innerWidth);
   const [columns] = React.useMemo(() => {
     const cProps = getComponentProps(data, unfilteredData, filters, selected, match, kindObj);
     const expandColumn = [];
-    if (expand) {
+    const cData = customData as CustomExpand;
+    if (expand && !cData?.disableRegularExpand) {
```

Comment:
curious question: was there some reason for the "regular expand logic" to not work fine? 🤔